site/e2e/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { Page } from "@playwright/test"
+
+export const buttons = {
+  starterTemplates: "Starter templates",
+  dockerTemplate: "Develop in Docker",
+  useTemplate: "Use template",
+  createTemplate: "Create template",
+  createWorkspace: "Create workspace",
+  submitCreateWorkspace: "Create workspace",
```

Comment:
Should this have a different label than the `createWorkspace` key above or does it not matter?

Reply:
The strings here are the text to find on the page. It just happens that we have two buttons with the same text.